### PR TITLE
Add support for skills network button; Remove Red Hat Runtimes and ICP4A support sections; Fix ruby dependencies

### PR DESCRIPTION
**Pull request by steven1046**
#### Were the changes tested on
- [x] Firefox (Desktop)
- [x] Safari (Desktop)
- [x] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] Dymanic Accessability Plugin (DAP)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: src/main/content/_assets/js/guide-static.js [hunk 1]
```diff
@@ -46,6 +46,28 @@
...
 
+    // Read skills network json to see if we need to link to a cloud hosted guide equivalent.
+    $.getJSON( "../../guides/guides-common/cloud-hosted-guides.json", function(data) {
+        var guide_name = window.location.pathname.replace('.html','').replace('/guides/', '');
+        var host = window.location.hostname;
+        var skills_network_url = host === "openliberty.io" ? data.skillNetworkUrl : data.stagingSkillNetworkUrl;
+        skills_network_url = skills_network_url.replace('{projectid}', guide_name);
+        if(data.guides.indexOf(guide_name) > -1){
+            $('.skills_network_description').text(data.tooltipText);
+            var skills_network_button = $('<a class="skills_network_button" target="_blank"></a>');
```
Comment (k8vance88) [inline src/main/content/_assets/js/guide-static.js]: You should rel="noopener noreferrer" for all target of _blank

**Review by k8vance88 — CHANGES_REQUESTED** [review]

File: src/main/content/_assets/js/guide-static.js (hunk 1 above)
Comment (mbroz2) [reply src/main/content/_assets/js/guide-static.js]: I think we have build code that adds that automatically for us?  @steven1046 can confirm

**Review by mbroz2 — DISMISSED** [review]
How'd we get staging out of sync from prod?  was something pushed to prod not from staging?

File: src/main/content/_assets/js/guide-static.js (hunk 1 above)
Comment (mbroz2) [reply src/main/content/_assets/js/guide-static.js]: confirmed that it's there on the staging site:
image

**Comment by steven1046** [comment]
The ruby gem stuff was delivered to prod as to not publish the skills network button but now i think that part can go in too because it doesn't show up unless the json is in guides-common in the prod/staging branches respectively.

**Comment by steven1046** [comment]
Yes, the build code can handle adding rel="noopener noreferrer", but it might be good to add it in the javascript file too so people looking at the file don't say the same thing in the future.

**Comment by steven1046** [comment]
Actually no, the build doesn't add rel="noopener noreferrer" to javascript...only html @mbroz2 so I will update this in draft/staging first.

**Comment by mbroz2** [comment]
And it doesn't help that I looked at completely the wrong piece of code.

**Review by k8vance88 — DISMISSED** [review]

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**